frameworks/hdfs/src/main/dist/svc.yml

```diff
@@ -251,11 +266,11 @@ pods:
               ./bootstrap --resolve-hosts name-$POD_INSTANCE_INDEX-node.$FRAMEWORK_HOST && ./{{HDFS_VERSION}}/bin/hdfs zkfc
         resource-set: zkfc-resources
         configs:
-          {{#KRB5_CONF_URI}}
+          {{#KERBEROS_ENABLED}}
           krb5-conf:
-            template: {{MESOS_SANDBOX}}/krb5.conf
+            template: krb5.conf
```

Comment:
Is this config file required for this task? It's not explicitly used in the command?

Reply:
Yeah it's still required because zkfc still needs to interact with the name node as its own component of HDFS and needs to be auth'ed for that communication to proceed successfully. krb5, as a result, needs to be in its JRE so it can talk to the KDC.